src/components/ha-dialog.ts

```diff
@@ -13,6 +14,7 @@ export const createCloseHeading = (hass: HomeAssistant, title: string) => html`
   <mwc-icon-button
     aria-label=${hass.localize("ui.dialogs.generic.close")}
     dialogAction="close"
+    ?rtl=${computeRTL(hass)}
```

Comment:
We now calculate `rtl` on every update, we might want to make a directive that checks if the language is changed.

Reply:
Also, we might want to switch to the official `dir` attribute instead of the `rtl` attribute.